### PR TITLE
Update maintainers

### PR DESCRIPTION
**Pull request by kujaku11**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,4 +49,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - jp
+    - kujaku11
+    - kkappler
+    - lheagy
```
Comment (jakirkham) [inline recipe/meta.yaml:54]: This reverts an accidental change [here]( https://github.com/conda-forge/mth5-feedstock/commit/ca8cf948fa01bfa3602eb738c8fd2faaac146c67#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR50-R52 )

File: recipe/meta.yaml [hunk 2]
```diff
@@ -49,4 +49,6 @@ about:
 
 extra:
   recipe-maintainers:
-    - jp
+    - kujaku11
+    - kkappler
```
Comment (jakirkham) [inline recipe/meta.yaml:53]: @kkappler are you ok being a maintainer here?

(asking as this a new maintainer that was not previously on the maintainer list)

Comment (kkappler) [reply recipe/meta.yaml:53]: @jakirkham I'm glad to be a maintainer - thanks for asking

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Pull request merged**